Planner plans trajectories that exceed acceleration limit.

### Commit
cb782c272ee6

### Steps to reproduce
How can the issue/bug be reproduced?
1. Switch into AUTO mode
2. Use RViz to plan a movement with velocity- and acceleration scaling 1.0.
3. Execute the movement.

### Expected behavior
Planning and Executing works without error.

### Observed behavior
Planning works, but executing fails. The robot doesn't even start moving.

### Logfiles
```
[ INFO] [1592834842.386682498]: Recovering successful
[ INFO] [1592834848.094568331]: Planning request received for MoveGroup action. Forwarding to planning pipeline.
[ INFO] [1592834848.094682587]: Initialized Point-to-Point Trajectory Generator.
[ INFO] [1592834848.094722985]: Generating PTP trajectory...
[ INFO] [1592834849.925210143]: Execution request received
[ERROR] [1592834849.972380926]: Acceleration limit violated by joint prbt_joint_6. Desired acceleration: 4.43134rad/s^2, limit:  4rad/s^2.
[ INFO] [1592834849.972646862]: Controller prbt/manipulator_joint_trajectory_controller/ successfully finished
[ WARN] [1592834849.972706115]: Controller handle prbt/manipulator_joint_trajectory_controller/ reports status PREEMPTED
[ INFO] [1592834849.972743617]: Completed trajectory execution with status PREEMPTED ...
[ INFO] [1592834849.972813891]: Execution completed: PREEMPTED
[ INFO] [1592834850.003672288]: PREEMPTED: Preempted
```
### Reference to test
https://github.com/PilzDE/pilz_robots/blob/melodic-devel/pilz_control/test/acceptance_test_acceleration_limit.md

--- a/prbt_support/config/manipulator_controller.yaml
+++ b/prbt_support/config/manipulator_controller.yaml
@@ -59,19 +59,19 @@ manipulator_joint_trajectory_controller:
   limits:
       prbt_joint_1:
         has_acceleration_limits: true
-        max_acceleration: 4.0
+        max_acceleration: 6.0
       prbt_joint_2:
         has_acceleration_limits: true
-        max_acceleration: 4.0
+        max_acceleration: 6.0
       prbt_joint_3:
         has_acceleration_limits: true
-        max_acceleration: 4.0
+        max_acceleration: 6.0
       prbt_joint_4:
         has_acceleration_limits: true
-        max_acceleration: 4.0
+        max_acceleration: 6.0
       prbt_joint_5:
         has_acceleration_limits: true
-        max_acceleration: 4.0
+        max_acceleration: 6.0
       prbt_joint_6:
         has_acceleration_limits: true
-        max_acceleration: 4.0
+        max_acceleration: 6.0

